### ISSUE TITLE
Support python2

### ISSUE THREAD

**Issue by nesl247** [astj/flake8-formatter-junit-xml#1]
Please add support for python2:

```python
Traceback (most recent call last):
  File "/Users/harrison/.local/share/virtualenvs/recommendation-feeds-6IquZfue/bin/flake8", line 11, in <module>
    sys.exit(main())
  File "/Users/harrison/.local/share/virtualenvs/recommendation-feeds-6IquZfue/lib/python2.7/site-packages/flake8/main/cli.py", line 16, in main
    app.run(argv)
  File "/Users/harrison/.local/share/virtualenvs/recommendation-feeds-6IquZfue/lib/python2.7/site-packages/flake8/main/application.py", line 396, in run
    self._run(argv)
  File "/Users/harrison/.local/share/virtualenvs/recommendation-feeds-6IquZfue/lib/python2.7/site-packages/flake8/main/application.py", line 385, in _run
    self.report()
  File "/Users/harrison/.local/share/virtualenvs/recommendation-feeds-6IquZfue/lib/python2.7/site-packages/flake8/main/application.py", line 379, in report
    self.formatter.stop()
  File "/Users/harrison/.local/share/virtualenvs/recommendation-feeds-6IquZfue/lib/python2.7/site-packages/flake8_formatter_junit_xml/formatter.py", line 31, in stop
    super().stop()
TypeError: super() takes at least 1 argument (0 given)
```

**Comment by astj** [comment]
Thank you for reporting.  I implemented tests and confirmed it in Travis. https://travis-ci.org/astj/flake8-formatter-junit-xml/jobs/322420296

**Comment by astj** [comment]
Now it passes. https://travis-ci.org/astj/flake8-formatter-junit-xml/jobs/322420957

**Comment by nesl247** [comment]
Could you actually create a new version for this? I'm not familiar with python, thought everything was coming from master since there are no releases. However that isn't the case.

**Comment by astj** [comment]
I'll release a new release to pypi before 2017 ends. please wait a while.

**Comment by astj** [comment]
Released as 0.0.3. https://pypi.python.org/pypi/flake8-formatter-junit-xml/0.0.3
